test/pgms/example_exceptions.py

```diff
@@ -0,0 +1,23 @@
+from cvc5_z3py_compat import *
+
+if __name__ == '__main__':
+    s = Solver()
+    s.set(**{"produce-models": "true"})
```

Comment:
```suggestion
    s.set("produce-models", True)
```
I've modified #71 to allow for this syntax as well.